### PR TITLE
Dreamchess: complete recipe now that it can be built

### PR DESCRIPTION
**Pull request by Lan72**
Created dreamchess.rdef.in file and upgrade dreamchess-0.2.0.recipe.

### CODE REVIEW

**Comment by diversys** [comment]
Please use a separate PR for mini_xml.

File: games-board/dreamchess/dreamchess-0.2.0.recipe [hunk 1]
```diff
@@ -22,31 +22,42 @@ LICENSE="GNU GPL v3"
...
-	lib:libsdl_image
-	lib:mxml
+	haiku$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libglu$secondaryArchSuffix
+	lib:libglew$secondaryArchSuffix
+	lib:libsdl$secondaryArchSuffix
+	lib:libsdl_image$secondaryArchSuffix
+	lib:libsdl_mixer$secondaryArchSuffix
+	lib:libmxml$secondaryArchSuffix
```
Comment (diversys) [inline games-board/dreamchess/dreamchess-0.2.0.recipe]: Sort alphabetically

File: games-board/dreamchess/dreamchess-0.2.0.recipe [hunk 2]
```diff
@@ -60,4 +71,20 @@ BUILD()
...
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local LONG_INFO="$SUMMARY"
+	sed \
+		-e "s|@APP_SIGNATURE@|$APP_SIGNATURE|" \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		-e "s|@LONG_INFO@|$LONG_INFO|" \
+		$portDir/additional-files/dreamchess.rdef.in > dreamchess.rdef
+
```
Comment (diversys) [inline games-board/dreamchess/dreamchess-0.2.0.recipe]: Move the the binary to $appsDir and rename it to DreamChess.

File: games-board/dreamchess/dreamchess-0.2.0.recipe [hunk 3]
```diff
@@ -22,31 +22,42 @@ LICENSE="GNU GPL v3"
 REVISION="1"
 SOURCE_URI="https://github.com/dreamchess/dreamchess/releases/download/$portVersion/dreamchess-$portVersion.tar.gz"
 CHECKSUM_SHA256="7615699995a635c3dff6c952aa0d32723c415c8bb42448b4269d3daa199d8059"
+SOURCE_DIR="dreamchess-$portVersion"
+ADDITIONAL_FILES="dreamchess.rdef.in"
 
 ARCHITECTURES="x86_gcc2"
+SECONDARY_ARCHITECTURES="?x86"
```
Comment (diversys) [inline games-board/dreamchess/dreamchess-0.2.0.recipe]: If the app builds with x86_gcc2 there is no need for SECONDARY_ARCHITECTURES.

File: games-board/dreamchess/dreamchess-0.2.0.recipe [hunk 4]
```diff
@@ -22,31 +22,42 @@ LICENSE="GNU GPL v3"
 REVISION="1"
 SOURCE_URI="https://github.com/dreamchess/dreamchess/releases/download/$portVersion/dreamchess-$portVersion.tar.gz"
 CHECKSUM_SHA256="7615699995a635c3dff6c952aa0d32723c415c8bb42448b4269d3daa199d8059"
+SOURCE_DIR="dreamchess-$portVersion"
+ADDITIONAL_FILES="dreamchess.rdef.in"
 
 ARCHITECTURES="x86_gcc2"
```
Comment (diversys) [inline games-board/dreamchess/dreamchess-0.2.0.recipe]: Does it build under x86_64?

Comment (Lan72) [reply games-board/dreamchess/dreamchess-0.2.0.recipe]: I do not know, I do not have access to x86_64.
On x86_gcc2 it is built and works, on x86 it is built, but it works unstably.

File: games-board/dreamchess/dreamchess-0.2.0.recipe (hunk 2 above)
Comment (diversys) [reply games-board/dreamchess/dreamchess-0.2.0.recipe]: Still stands.

**Comment by scottmc** [comment]
@Begasus can you try building this one and see if it works on both architectures?

**Comment by korli** [comment]
Applied in 000203951900520192922becb4e3b79ceca2132e

**Pull request closed without merging**